client/src/nl/sensorlab/videowall/animation/baseanimations/sorting/Sort.java

```diff
@@ -0,0 +1,47 @@
+package nl.sensorlab.videowall.animation.baseanimations.sorting;
+import processing.core.PGraphics;
+
+public abstract class Sort {
+	protected int[] valuesArray;
+	public int currentSelected = 0;
+	public int num;
+
+	public Sort(int[] _inputValuesArray) {
+		this.valuesArray = _inputValuesArray;
+	}
+
+	public abstract void sortStep();
+
+	public void draw(PGraphics g, int _width, int _height, float _maxValue, int _color) {
```

Comment:
Please use Java notation for variables, ```_variableName``` is C/C++ style notation

Reply:
Done